voucher/src/test/java/com/adyen/checkout/voucher/internal/ui/DefaultVoucherDelegateTest.kt

```diff
@@ -33,6 +35,7 @@ internal class DefaultVoucherDelegateTest {
         delegate = DefaultVoucherDelegate(
             ActionObserverRepository(),
             GenericComponentParamsMapper(null, null).mapToParams(configuration, null),
+            PdfOpener()
```

Comment:
This should be a mock, because if this instance is called in a test it will crash